example/iam/config.yml

```diff
@@ -255,3 +255,26 @@ mboyd-int:
 
 pq:
   prefix: pq
+
+mboyd-ds-cumulus:
```

Comment:
This probably doesn't need to be committed. This was for deploying my stack to Devseed's AWS